### DISCUSSION TITLE
pilot.wp.pl

### DISCUSSION THREAD

**Discussion by MESWEB** [MajkiIT/polish-ads-filter#17306] [Q&A]
Czy da sie ogladac pilot.wp.pl z uBlockiem? Mi wyskakuje ciagle alert o wykryiu adBlocka

**Comment by hawkeye116477** [comment]
Nie

**Comment by MESWEB** [comment]
@xxcriticxx kolego doceniam bardzo Twoje nieudolne starania. Jak widzisz kolega wyzej odpisal wiadomosc "nie" na odpier...ol nie wyjasniajac niczego i nie posiadajac zadnego argumentu. Natomiast Ty kolego wbijasz do mojego pytania i bezczelnie ustawiasz odpowiedz miernoty jako rozwiazanie. Domniemam iz te dwa konta sa jednej osoby a jesli sie myle to zapewne sobie razem "trzymacie" przy sikaniu.

**Reply by xxcriticxx** [reply]
@MESWEB wow smiechu warta ta twoja wypowiedz. Ide do kibla zrobic siusiu.

**Comment by hawkeye116477** [comment]
Grzeczniej trochę... Obecnie po prostu się nie da oglądać z uBO, wyskakuje komunikat, tyle w temacie. 
Chcesz jakiegoś rozwiązania to od tego jest zakładka „issues". Dla twojej wiadomości, te dwa konta nie są jednej osoby.

**Comment by krystian3w** [comment]
Jak chodzi o chrome, to trzeba wyłączyć arsenał Olega/Majkiego:

```
#@#+js(set-constant, wp_abtest, null)
#@#+js(set-constant, Number.isNaN, trueFunc)
#@#+js(set-constant, PWA_adbd, 2)
```

i po aktualizacji uBO do 1.30.4 włączyć AdGuard Base (namierzenie czym AdGuard naprawia to przebicie się przez jakieś 200-300 filtrów).

Nie wiem czy fix zadziała z uBO 1.27.10 / uBO 1.28.4 / uBO 1.29.0 / uBO 1.29.2.
